Frontend/CO.CDP.OrganisationApp/WebApiClients/OrganisationExtensions.cs

```diff
@@ -20,6 +20,11 @@ public static bool IsPendingBuyer(this Organisation.WebApiClient.Organisation or
         return organisation.Details.PendingRoles.Contains(PartyRole.Buyer);
     }
 
+    public static bool IsRejectedBuyer(this Organisation.WebApiClient.Organisation organisation)
+    {
+        return IsPendingBuyer(organisation) && organisation.Details.Approval.ReviewedBy != null;
+    }
+
```

Comment:
Inferring rejected status by the fact that the buyer is pending _and_ has a non null review. Would be nice to have the review_status field explicitly recording this instead of having to infer it.

Reply:
This decision will be done on the API side where we will expose the status on the response.